ERCS/erc-7970.md

```diff
@@ -0,0 +1,215 @@
+|-------------|--------------------------------|-----------------------------------------|
+| 0           | Connection request             | Public key + encrypted request body     |
+| 1           | Connection-request response    | Public key + encrypted response body    |
+| 2           | Encrypted text message         | AES-GCM encrypted message data          |
+
+
+Message types 0, 1, and 2 are reserved to promote basic interoperability, but implementers MAY define and use other message types (3 and above) for custom behavior.
+
+These types are intended to follow this general RECOMMENDED cryptographic model:
+
+- A shared symmetric key is derived using Elliptic Curve Diffie-Hellman (ECDH) from the sender and recipient key pairs.
+- Payloads are encrypted using an authenticated encryption scheme, typically AES-GCM.
+
+Implementers MAY vary the details of symmetric key derivation (e.g., direct use of shared secret, use of HKDF, or key rotation strategies) and encryption parameters (e.g., nonce generation), so long as the overall structure and intent are preserved. However, if substantial deviations from ECDH-based key exchange or AES-GCM semantics are used, it is RECOMMENDED to define a new messageType (e.g., >= 3) to avoid misinterpretation by other clients.
+
+All encryption, decryption, and key exchange logic MUST be handled off-chain.
+
+## Rationale
+
+This core protocol is intended for encrypted communication between two addresses. Multiple message types can be filtered using `eth_getLogs` or `eth_subscribe` without needing contract storage. The core protocol omits economic interactions and access controls. By emitting only a single event, the protocol reduces on-chain complexity and minimizes gas fees.
+
+### Related EIPs
+
+This proposal is intentionally minimal and stateless, in the spirit of [ERC-3722: Poster](./eip-3722.md), which emits plaintext posts for decentralized social media. While Poster focuses on public broadcast-style messaging, this proposed standard defines a private, encrypted peer-to-peer communication model.
+
+The two differ not just in use case but in security assumptions: posting plaintext messages to an immutable, public blockchain poses long-term risks, especially if illegal or harmful content becomes permanently associated with a project or platform. This standard is specifically intended for encrypted communication between two addresses - not plaintext public posts.
+
+This proposal also contrasts with [ERC-7627: Secure Messaging Protocol](./eip-7627.md), which defines a comprehensive messaging interface, including on-chain key management, encryption algorithm enumeration, and session identifiers.
+
+By contrast, this standard is intended to facilitate basic encrypted communication between two addresses within a wide range of app environments, while offering the flexibility to incorporate other features through app-specific use of the `messageType` parameter.
```

Comment:
Comparing this proposal as a whole to other proposals likely belongs in the Motivation section. The Rationale section should be used to explain technical choices made within the proposal itself.